modules/ossm-deploy-multi-mesh.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+
+[id="ossm-deploy-multi-mesh_{context}"]
+= Multimesh or Federated deployment model
+
+[role="_abstract"]
+*Federation* is a deployment model that lets you share services and workloads between separate meshes managed in distinct administrative domains.
+
+The Istio multi-cluster model requires a high level of trust between meshes, and requires remote access to all Kubernetes API servers on which the individual meshes reside. {ProductName} federation takes an opinionated approach to a multi-cluster implementation of Service Mesh that assumes _minimal_ trust between meshes.
```

Comment:
...federation takes an opinionated approach... Is this a common and widely-known way of describing it? Maybe I'm not aware of the context and hence don't quite understand the meaning of it...